_layouts/news-grid.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+
+    <main id="main-content">
+       
+      <section class="usa-card__header bg-blue-80v">
+            <div class="grid-container">
+                <h1 class="font-heading-3xl text-accent-cool">{{ page.title }}</h1>
+                <h2 class="font-alt-xl text-base-lightest tablet:grid-col-9">{{ page.description }}</h2>
+            </div>
+        </div>
+        </section>
+       
+ <section class="grid-container margin-top-4 margin-bottom-4 border-bottom border-base-light">
+    <p class="usa-intro mobile:font-heading-lg tablet:grid-col-9">
+		{{ page.intro }}
+	</p>
+ </section>
+ 
+ <section class="grid-container padding-y-2">
+  <div class="grid-row grid-gap">
+    <ul class="usa-card-group">
+      {% for article in page.articles %}
+ <li class="tablet:grid-col-4 usa-card">
+    <div class="usa-card__container">
+      <header class="usa-card__header">
+        <h6 class="text-base-dark">{{ article.source }}</h6>
+        <h2 class="usa-card__heading">{{ article.title }}</h2>
+      </header>
+      <div class="usa-card__media">
+        <div class="usa-card__img">
+          <img
+            src="{{ article.image-source }}"
+            alt="{{ article.image-alt }}"
+            class="minh-card maxh-card"
+          />
+        </div>
+      </div>
+      <div class="usa-card__body">
+        <p>
+          {{ article.description }}
+        </p>
+      </div>
+      <div class="usa-card__footer">
+        <a href="{{ article.link-source }}" class="usa-link link-arrow-right">{{ article.link }}</a>
```

Comment:
Something to consider for accessibility is to avoid sequential links with the same link text. (Because when you tab through, it just sounds like 'read more... read more... read more...' to someone using a screenreader. 

As an alternative, you could use link the card title instead. If you want to keep the customizable link text, I would suggest including some examples of how you might vary those instead of entering the same text for all of them.